src/Schema/SnapshotStoreSchema.php

```diff
@@ -32,13 +32,17 @@ public static function create(Schema $schema, $snapshotName = 'snapshot')
     {
         $snapshot = $schema->createTable($snapshotName);
 
+        // UUID4 of linked aggregate
+        $snapshot->addColumn('aggregate_id', 'string', ['fixed' => true, 'length' => 36]);
+        // Class of the linked aggregate
         $snapshot->addColumn('aggregate_type', 'string');
-        $snapshot->addColumn('aggregate_id', 'string');
-        $snapshot->addColumn('last_version', 'integer');
-        $snapshot->addColumn('created_at', 'string');
+        // Version of the aggregate after event was recorded
+        $snapshot->addColumn('last_version', 'integer', ['unsigned' => true]);
+        // DateTime ISO8601 + microseconds UTC stored as a string e.g. 2016-02-02T11:45:39.000000
+        $snapshot->addColumn('created_at', 'string', ['fixed' => true, 'length' => 26]);
         $snapshot->addColumn('aggregate_root', 'blob');
 
-        $snapshot->addIndex(['aggregate_type', 'aggregate_id']);
+        $snapshot->addIndex(['aggregate_id', 'aggregate_type']);
```

Comment:
We definitely need a partial index for column aggregate_type e.g. length of 20 for performance reasons.